topics/07_web_ui_testing/ui_test_text_box.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from selenium import webdriver
+from ui_text_box_page import TextBoxPage
+
+
+@pytest.fixture
+def driver():
+    """Fixture to set up"""
+    driver = webdriver.Chrome()
+    driver.maximize_window()
+    yield driver
+    driver.quit()
+
+
+def test_text_box_form_submission(driver):
+    """Tests validation"""
+    page = TextBoxPage(driver)
+    page.open()
+
+    """Test data"""
+    full_name = "Donald Duck"
```

Comment:
The test data could be better organized using a data structure like a dictionary or a data class or placed in a JSON file. This can help make the code cleaner and more maintainable, especially when there are more data fields.